### PR TITLE
JIRA: VZ-9263 remove clustermonitorgraph CRD and associated code

### PR DESCRIPTION
**Pull request by jmaron99**
- Removed clustermonitorgraph CRD
- cluster_graph_handler was removed
- project_graph_handler and handler_utils updated with shared code that was in cluster_graph_handler

This CRD was related to the Rancher Monitoring App which is currently hidden and is designated for removal

### CODE REVIEW

File: pkg/api/norman/customization/monitor/handler_utils.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ import (
 
 const (
 	prometheusReqTimeout = 30 * time.Second
+	projectIDAnn         = "field.cattle.io/projectId"
```
Comment (prasad-shirodkar) [inline pkg/api/norman/customization/monitor/handler_utils.go:24]: Why is this added?

Comment (jmaron99) [reply pkg/api/norman/customization/monitor/handler_utils.go:24]: Because it was coded in the cluster_graph_handler even though this const is leveraged by this and the project handler module.  I have removed the cluster graph handler since it's associated CRD no longer exists.

**Review by prasad-shirodkar — DISMISSED** [review]

**Review by prasad-shirodkar — APPROVED** [review]
